### PR TITLE
Math magicians: Add events

### PR DESCRIPTION
**Pull request by oliverSCZ**
I did the following:

- Created a new branch called feature_events
- Used correct Gitflow
- Effectively set up a React linter.
- Developed just the UI of the calculator
- Imported  files in Calculator component.
- Implemented the event handlers need to use the math logic from calculate.js and operate.js in React component.
- Tested app in the browser.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    return one.mod(two).toString();
+  }
+  throw Error(`Unknown operation '${operation}'`);
+}
```
Comment (enaburekhan) [inline src/logic/operate.js:26]: - [ ] I will suggest you add the try-catch block here. Just like the other `if/else` statements. Something like this:

```
if (operation === '%') {
     try {
      return one.mod(two).toString();
    } catch (err) {
      return "Can't find modulo as can't divide by 0.";
    }
  }
  throw Error(`Unknown operation '${operation}'`);
}
```

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,33 +1,173 @@
 /* eslint-disable react/prefer-stateless-function */
 import React, { Component } from 'react';
 import './calculator.css';
+import calculate from '../logic/calculate';
+import '../logic/operate';
```
Comment (enaburekhan) [inline src/components/Calculator.js]: **Optional**

- [x] I will suggest you remove this line. operate.js is already being used by calculate.js, so there's no need to import it.

**Review by enaburekhan — CHANGES_REQUESTED** [review]
Hi OliverSCZ, 

Good job so far!

- [ ] Major highlights

- The PR summary is very descriptive
- Passed linters check

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] It be nice to add a descriptive readme to the App. 

_Check the comments under the review._


## Optional suggestions

- [ ] It will be nice to add commit messages frequently.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by omar-labana** [comment]
Hi @oliverSCZ, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**